Xamarin.Forms.Platform.Android/AppCompat/PickerRenderer.cs

```diff
@@ -140,7 +148,8 @@ void OnClick()
 				{
 					builder.SetTitle(model.Title ?? "");
 					string[] items = model.Items.ToArray();
-					builder.SetItems(items, (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
+					//builder.SetItems(items, (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
+					builder.SetSingleChoiceItems(items, (int)Element.GetValue(Picker.SelectedIndexProperty), (s, e) => ((IElementController)model).SetValueFromRenderer(Picker.SelectedIndexProperty, e.Which));
```

Comment:
how is this change related to Placeholder properties ?

Reply:
Ah sorry, this shouldn't have been in. I was playing around with some other code.